examples/basics/logistic.py

```diff
@@ -3,7 +3,7 @@
 import minpy.numpy.random as random
 import minpy.dispatch.policy as policy
 
-#np.set_policy(policy.OnlyNumPyPolicy())
+np.set_policy(policy.OnlyNumPyPolicy())
```

Comment:
Sorry. I don't have GPU MXNet on my box. I'll revert this change.
Normally we shouldn't commit changes like this (changing file-wise policy or file-wise logging options).
My bad.